modules/apps/feature-flag/feature-flag-test/src/testFunctional/tests/FeatureFlagDeprecation.testcase

```diff
@@ -0,0 +1,56 @@
+@component-name = "portal-feature-flag"
+definition {
+
+	property custom.properties = "feature.flag.LPS-162766=true${line.separator}feature.flag.LPS-162766.description=Test Deprecation Description 404 sample page${line.separator}feature.flag.LPS-162766.title=Test Deprecation Title 404 sample page${line.separator}feature.flag.LPS-162766.type=deprecation${line.separator}feature.flag.ui.visible[deprecation]=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Feature Flag";
+	property testray.main.component.name = "Feature Flag";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given a feature flag deprecation property is set") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			PortalSettings.gotoConfiguration(
+				configurationCategory = "Feature Flags",
+				configurationName = "Deprecation",
+				configurationScope = "Virtual Instance Scope");
+		}
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-189292 - This test asserts that a deprecated feature flag can be utilized."
+	@priority = 5
+	test CanBeUtilized {
+		property portal.acceptance = "true";
+
+		task ("When enable deprecation feature flag at Instance Settings") {
+			Check.checkToggleSwitch(
+				locator1 = "FeatureFlag#TOGGLE_ROW",
+				title_row = "Test Deprecation Title 404 sample page",
+				toggle_state = "Disabled");
+		}
+
+		task ("Then the feature can be utilized") {
+			AssertElementPresent(
+				locator1 = "FeatureFlag#TOGGLE_ROW",
+				title_row = "Test Deprecation Title 404 sample page",
+				toggle_state = "Enabled");
+		}
```

Comment:
Not sure if `AssertElementPresent` is enough for asserting the FF can be utilized, Thiago proposed to also refresh the page and assert it again to catch any bug from the back-end side. What do you think @john-co ?

Reply:
This is not enough. If you're using LPS-162766 as the sample feature flag, then you can assert the basic behavior of the feature. You review the basic feature in the story/test scenarios ticket.